pkg/statsd/handler_http_forwarder_v2.go

```diff
@@ -156,6 +160,7 @@ func NewHttpForwarderHandlerV2(
 		consolidatedMetrics:   ch,
 		client:                httpClient.Client,
 		headers:               headers,
+		dynHeaders:            dynHeaders,
```

Comment:
`dynHeaders` should be pre-baked to remove the `tagName+":"` in the loop.

- [ ] this

Reply:
This only includes the header/tag names for extracting values from tags. As explained in another comment, the dynamic header values cannot be pre-baked when gostatsd starts. Or maybe I missed your point?

Reply:
Ahh, the `dynName +":"` part can definitely be optimized. 👍

Reply:
I've pushed a commit to pre-bake the colon.